go.mod

```diff
@@ -14,31 +14,31 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.14.0
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.7.0
 	github.com/aws/aws-sdk-go-v2/credentials v1.17.27
-	github.com/cyphar/filepath-securejoin v0.3.1
+	github.com/cyphar/filepath-securejoin v0.3.2
 	github.com/dimchansky/utfbom v1.1.1
 	github.com/fluxcd/cli-utils v0.36.0-flux.9
 	github.com/fluxcd/kustomize-controller/api v1.3.0
 	github.com/fluxcd/pkg/apis/acl v0.3.0
-	github.com/fluxcd/pkg/apis/event v0.10.0
-	github.com/fluxcd/pkg/apis/kustomize v1.6.0
-	github.com/fluxcd/pkg/apis/meta v1.6.0
-	github.com/fluxcd/pkg/http/fetch v0.12.0
+	github.com/fluxcd/pkg/apis/event v0.10.1
+	github.com/fluxcd/pkg/apis/kustomize v1.6.1
+	github.com/fluxcd/pkg/apis/meta v1.6.1
+	github.com/fluxcd/pkg/http/fetch v0.12.1
 	github.com/fluxcd/pkg/kustomize v1.13.0
-	github.com/fluxcd/pkg/runtime v0.49.0
-	github.com/fluxcd/pkg/ssa v0.41.0
-	github.com/fluxcd/pkg/tar v0.8.0
+	github.com/fluxcd/pkg/runtime v0.49.1
+	github.com/fluxcd/pkg/ssa v0.41.1
+	github.com/fluxcd/pkg/tar v0.8.1
 	github.com/fluxcd/pkg/testserver v0.7.0
-	github.com/fluxcd/source-controller/api v1.3.0
+	github.com/fluxcd/source-controller/api v1.4.0
```

Comment:
This needs to be in sync with https://github.com/fluxcd/kustomize-controller/blob/aabb41fc5d8ea91b2980f6fc6c1066f82ec148e5/config/default/kustomization.yaml#L5 please update it in a new commit